dev/kyuubi-extension-spark-common/src/main/scala/org/apache/kyuubi/sql/watchdog/ForcedMaxOutputRowsBase.scala

```diff
@@ -70,14 +65,14 @@ trait ForcedMaxOutputRowsBase extends Rule[LogicalPlan] {
         true
       }
     case _: MultiInstanceRelation => true
+    case _: Join => true
     case _ => false
   }
 
   protected def canInsertLimit(p: LogicalPlan, maxOutputRowsOpt: Option[Int]): Boolean = {
     maxOutputRowsOpt match {
       case Some(forcedMaxOutputRows) => canInsertLimitInner(p) &&
-          !p.maxRows.exists(_ <= forcedMaxOutputRows) &&
-          !isView
```

Comment:
why we need this change ?

Reply:
SQL with view will first enter the resolveViews during analyze.The Analyzer's Execute method will be called again, and the analysis of SQL statements with view will recursively call Analyzer's Batches. https://github.com/apache/incubator-kyuubi/pull/1247 add "isView" fixed this.
![image](https://user-images.githubusercontent.com/18713676/153526263-8d27902e-831f-4f9b-983a-b283c00eda0c.png)

But now Place the ForcedMaxOutputRowsRule in the projectOptimizerRule stage,we do not need this.